proposals/esm-integration/README.md

```diff
@@ -125,20 +125,21 @@ Some imports are challenging to handle in the current WebAssembly specification.
 
 ### wasm imports <- JS exports
 
-| export type | value (not a WebAssembly.Global)* | global | memory | table | function |
-|-------------|------------------|--------|--------|-------|----------|
-|              | Error                         | Error  | Error  | Error | snapshot |
-
-\*While WebAssembly only has the concept of globals, JS could export either a regular JS value or a `WebAssembly.Global`.
+| import type | global | memory | table | function     |
```

Comment:
`export`?

Reply:
import type seems more appropriate here--the path is chosen based on how the WebAssembly module declares its imports. I really appreciate this fix.